sass/worker_protocol.proto

```diff
@@ -0,0 +1,51 @@
+// Copyright 2015 The Bazel Authors. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
```

Comment:
I'm guessing this isn't the canonical source of this protobuf definition—where is that? Can we depend on it from there?

Reply:
The current impl of worker reads the local package to load the proto definition, which is why we need to copy this proto here as well.

I believe the canonical source is https://github.com/bazelbuild/bazel/blob/master/src/main/protobuf/worker_protocol.proto

I am open to suggestions about how to depend this and resolve the path accordingly.